test/externalTests/elementfi.sh

```diff
@@ -89,6 +89,8 @@ function elementfi_test
     # TODO: Remove when https://github.com/element-fi/elf-contracts/issues/243 is fixed.
     sed -i 's|^\s*require(_expiration - block\.timestamp < _unitSeconds);\s*$||g' contracts/ConvergentCurvePool.sol
 
+    find . -name "*.sol" -exec sed -i -e 's/^\(\s*\)\(assembly\)/\1\/\/\/ @solidity memory-safe-assembly\n\1\2/' '{}' \;
```

Comment:
This one actually *isn't* memory-safe... and apparently I was fooled by our test cases in that just inlining unconditionally will just work and not break anything :-(.